adapters/repos/db/shard_dimension_tracking.go

```diff
@@ -17,36 +17,12 @@ import (
 	"github.com/weaviate/weaviate/adapters/repos/db/lsmkv"
 	"github.com/weaviate/weaviate/adapters/repos/db/vector/common"
 	"github.com/weaviate/weaviate/cluster/usage/types"
+	"github.com/weaviate/weaviate/entities/dimensioncategory"
 	schemaConfig "github.com/weaviate/weaviate/entities/schema/config"
 	hnswent "github.com/weaviate/weaviate/entities/vectorindex/hnsw"
 	"github.com/weaviate/weaviate/usecases/monitoring"
 )
 
-type DimensionCategory int
```

Comment:
I feel strongly that this PR should not touch `shard_dimension_tracking` at all as this then becomes a very high risk change (changing billing logic direct to a stable branch), we will need additional QA, and it will make back porting billing changes harder.

Additional using the same enum for billing dimension category vs the default quantization means we can't support things like `DEFAULT_QUANTIZATION=rq1` for 1 bit quantization.

Reply:
I would prefer if we only have one definition of each quantisation in the code.  Maintaining the code becomes more challenging if there are e.g. two CompressionRQ definitions, and we have to convert between them.

Billing for extra options is fine, we just add a calculation for the new optioin "rq1", and it's ok if it is the same calculation as "rq"